### PR TITLE
Add support to search anything.

### PR DESCRIPTION
**Pull request by ryanpcmcquen**
Rebase of: https://github.com/farcaller/DashDoc/pull/19

@farcaller, does this look OK?

### CODE REVIEW

File: DashDoc.py [hunk 1]
```diff
@@ -68,3 +68,16 @@ def run(self, edit, flip_syntax_sensitive=False, run_in_background=False):
...
+        self.window.show_input_panel('Topic?', '', self.on_done, self.on_change, self.on_cancel)
+
+    def on_change(self, name):
+        pass
+
+    def on_cancel(self, name):
+        pass
+
+    def on_done(self, topic):
+        subprocess.call(["open", "dash://" + topic])
```
Comment (farcaller) [inline DashDoc.py]: I'd rather have this call to the [dash-plugin://](https://kapeli.com/dash_plugins) url too. note that you need to quote the query.

Comment (ryanpcmcquen) [reply DashDoc.py]: Thank you @farcaller, will fix!

**Comment by ryanpcmcquen** [comment]
@farcaller, how do the changes look?

File: Default.sublime-commands [hunk 2]
```diff
@@ -28,5 +28,9 @@
             "flip_syntax_sensitive": true,
             "run_in_background": true
         }
+    },
+    {
+        "caption": "DashDoc: search any topic",
+        "command": "dash_doc_search"
     }
-]
+]
```
Comment (farcaller) [inline Default.sublime-commands]: Fix the missing newline please.

**Comment by ryanpcmcquen** [comment]
@farcaller, resolved!

**Comment by farcaller** [comment]
Thanks!

**Comment by farcaller** [comment]
Tagged as 2.7.0

**Pull request merged**